libraries/AC_AttitudeControl/AC_AttitudeControl_Heli.cpp

```diff
@@ -283,56 +283,27 @@ void AC_AttitudeControl_Heli::update_althold_lean_angle_max(float throttle_in)
 // rate_bf_to_motor_roll_pitch - ask the rate controller to calculate the motor outputs to achieve the target rate in radians/second
 void AC_AttitudeControl_Heli::rate_bf_to_motor_roll_pitch(const Vector3f &rate_rads, float rate_roll_target_rads, float rate_pitch_target_rads)
 {
-    float roll_pd, roll_i, roll_ff;             // used to capture pid values
-    float pitch_pd, pitch_i, pitch_ff;          // used to capture pid values
-    float rate_roll_error_rads, rate_pitch_error_rads;    // simply target_rate - current_rate
+    float roll_pid, roll_ff;             // used to capture pid values
```

Comment:
it is better to directly assign value on object creation in cpp as it will use the object constructor to assign the value instead of doing it in two steps : varaible create then assignement !

Reply:
I am maintaining consistency with the way Heli has been done here already. I would be happy to work with Bill and Chris to do more though.